tests/tpot_tests.py

```diff
@@ -335,16 +348,38 @@ def test_timeout():
     tpot_obj._optimized_pipeline = creator.Individual.from_string(pipeline_string, tpot_obj._pset)
     tpot_obj.fitted_pipeline_ = tpot_obj._toolbox.compile(expr=tpot_obj._optimized_pipeline)
     # test _wrapped_cross_val_score with cv=20 so that it is impossible to finish in 1 second
+    cv = model_selection.KFold(n_splits=20).split(
+        X=training_features_r,
+        y=training_target_r,
+    )
+    cv = model_selection._split.check_cv(cv, training_target_r, classifier=False)
     return_value = _wrapped_cross_val_score(tpot_obj.fitted_pipeline_,
                                             training_features_r,
                                             training_target_r,
-                                            cv=20,
+                                            cv=cv,
                                             scoring_function='neg_mean_squared_error',
                                             sample_weight=None,
                                             groups=None,
                                             timeout=1)
     assert return_value == "Timeout"
 
+def test_custom_cv_test_generator():
```

Comment:
New test for this specific bug. Not very fancy, just pass a couple of custom cv generators and make sure there are no errors.

Reply:
OK, it look goods